### PR TITLE
Add README & LICENSE template files into a Kentico repository template

### PR DESCRIPTION
**Pull request by Chleba**
### Motivation

The template for new Kentico repositories has been a little bit shallow.
There was no README.md and LICENSE.md file with a basic template for the user's information.
I took the liberty to put them into our template.

### Checklist

- [X] Code follows coding conventions held in this repo
- [X] Automated tests have been added - no tests needed
- [X] Tests are passing - no tests needed
- [X] Docs have been updated (if applicable)
- [X] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

- Check if the README.md file exists, the file is not empty and has valid information.
- Check if the LICENSE.md file exists, the file is not empty and has valid information.

### CODE REVIEW

File: LICENSE.md
Comment (Simply007) [inline LICENSE.md]:
```suggestion
Copyright (c) [Current year here] Kentico
```

File: README.md
Comment (Simply007) [inline README.md:68]: Do we want to extract links to the end?

File: README.md
Comment (Simply007) [inline README.md]: Do we want to use this hacky way of scoling? 

If so, we should use #readme anchor

File: README.md
Comment (Simply007) [inline README.md:1]: I would align these with https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#badges

And add a comment link with that.

File: README.md
Comment (Simply007) [inline README.md]: We can use one of our repo as a reference. I.E home repository/repo template/...

File: README.md
Comment (Simply007) [inline README.md]:
```suggestion
Distributed under the MIT License. See [`LICENSE.md`](./LICENSE.md) for more information.
```

**Review by Simply007 — CHANGES_REQUESTED** [review]
The idea is great. It basically.

We can add some information about_
* https://github.com/Kentico/Home/wiki
    * and then specifically mention https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project

~Some parts are discutable - let's go through them together.~

I would add a CODEOWNERS + CI template file too.

CODEOWNERS

```plain
# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
# See https://help.github.com/articles/about-code-owners/

* [Maintainer team https://github.com/Kentico/Home/wiki/Guidelines-for-GitHub-permissions-in-the-Kentico-organization]
```

CI template `.github/workflows/build.yml`

```yml

# For more information do to https://github.com/Kentico/Home/wiki/CI-and-Automation-Guidelines

name: "Build"   # Represents the name of the whole action.
on:   # Specifies the section where we describe our build triggers.
  push: # The action runs with push.
  pull_request: # The action runs with a pull request.
  schedule: # Specifies the section where we describe the schedule of running the action.
    - cron: '0 18 * * 1' # The CRON expression describes when to run the action.
jobs: # Specifies the section where we describe our jobs.
  Build: # Specific job section.
    runs-on: ubuntu-latest   # Describes the environment.
    steps: # Specifies the section where we describe the job's steps.
    - name: Output some string
      run: echo "Set some actual CI steps"
head.

```

---

Once merged - link the repo template to the sections as https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#readme-documentation + https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#license directly

File: README.md
Comment (Simply007) [reply README.md]: Let's remove all of these.

File: README.md
Comment (Simply007) [reply README.md:1]: At least Kontent discord badge (and unify styles).

File: README.md
Comment (Simply007) [reply README.md]: Let's use Home repo

File: README.md
Comment (Chleba) [reply README.md:68]: All stored values in variables would be in one place and it's like a more readable list.

File: README.md
Comment (Chleba) [reply README.md]: All removed

File: .github/workflows/build.yml
Comment (Simply007) [inline .github/workflows/build.yml]:
```suggestion
```

forgottent word

**Review by Simply007 — APPROVED** [review]
Just one last thing I mention (I have used verified commit to fix it).

Juts before merge - add the repo template links to the sections as https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#readme-documentation + https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#license directly

**Comment by Simply007** [comment]
@Chleba 
Please add links to https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#readme-documentation + https://github.com/Kentico/Home/wiki/Checklist-for-publishing-a-new-OS-project#license

**Comment by Chleba** [comment]
Home Wiki is updated with corresponding Links to README.md and LICENSE.md.

**Comment by Simply007** [comment]
> Home Wiki is updated with corresponding Links to README.md and LICENSE.md.

Thx

**Pull request merged**
